### PR TITLE
Grid and slider

### PR DESCRIPTION
**Pull request by rahulyadav139**
grid and slider components are created.

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Code seems good to me.

**Review by saurabhchirde — APPROVED** [review]
Seems good to me, you can proceed to merge.

**Review by j24m — APPROVED** [review]
Perfect. GTM! :)

File: app.js [hunk 1]
```diff
@@ -56,18 +56,18 @@ btnCodeSnippet.forEach(el => {
...
-      .replace(/</g, '&lt;')
-      .replace(/>/g, '&gt;')
-      .replace(/"/g, '&quot;')
-      .replace(/'/g, '&#039;')
-      .trim();
-    hljs.highlightElement(el);
-  });
-});
+// document.addEventListener('DOMContentLoaded', event => {
+//   document.querySelectorAll('pre code').forEach(el => {
```
[the remark this answers is not in the document]
Comment (rahulyadav139) [reply app.js:60]: Thank you for the suggestation. I'll look into it.

**Pull request merged**